3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,218 @@
+//Java-scipt Week3 Homework//
+
+//(1) - Item array removal// 
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  const filteredNames = names.filter(name => name !== nameToRemove);
+  
+  console.log(filteredNames); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+  
+
+  //When will we be there??//
+
+  function calculateTravelTime(travelInformation) {
+    const { speed, destinationDistance } = travelInformation;
+    const totalTimeInHours = destinationDistance / speed;
+    const hours = Math.floor(totalTimeInHours);
+    const minutes = Math.round((totalTimeInHours - hours) * 60);
+  
+    const formattedTime = `${hours} hours and ${minutes} minutes`;
+  
+    return formattedTime;
+  }
+  
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  const travelTime = calculateTravelTime(travelInformation);
+  console.log(travelTime);
+
+
+  //(2) - Series duration of my life// 
+
+  const seriesDurations = [
+    {
+      title: "Money Heist",
+      days: 1,
+      hours: 23,
+      minutes: 50,
+    },
+    {
+      title: "El Chapo",
+      days: 1,
+      hours: 5,
+      minutes: 20,
+    },
+    {
+      title: "The Wire",
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    },
+  ];
+  
+  function logOutSeriesText() {
+    const totalMinutesInLifespan = 80 * 365 * 24 * 60; // 80 years in minutes
+    let totalMinutesWatchingSeries = 0;
+  
+    for (const series of seriesDurations) {
+      const seriesMinutes = series.days * 24 * 60 + series.hours * 60 + series.minutes;
+      totalMinutesWatchingSeries += seriesMinutes;
+      const percentageOfLife = (seriesMinutes / totalMinutesInLifespan) * 100;
+      console.log(`${series.title} took ${percentageOfLife.toFixed(3)}% of my life`);
+    }
+  
+    const totalPercentage = (totalMinutesWatchingSeries / totalMinutesInLifespan) * 100;
+    console.log(`\nIn total that is ${totalPercentage.toFixed(3)}% of my life`);
+  }
+  
+  logOutSeriesText();
+  
```

Comment:
🌟 Brilliant use of `constants` like `totalMinutesInLifespan` to make the calculations more readable. The way you've tackled percentage calculations for each series is commendable.
💡 Suggestion: It would be good to consider handling cases where the `seriesDurations` array might be `empty`.